### ISSUE TITLE
Installer shows deprecated messages on PHP 8.1.2

### ISSUE THREAD

**Issue by browniebraun** [Cacti/cacti#5033]
# Describe the bug

Clean installation of 1.2.x branch, version 1.2.23, shows deprecated messages directly when the install wizard starts.

```
Action failed
Sorry, we could not process your last action.
Error:	200 OK
Reason:	
Deprecated: escapeshellcmd(): Passing null to parameter #1 ($command) of type string is deprecated in /var/www/html/cacti/lib/functions.php on line 4473

Deprecated: file_exists(): Passing null to parameter #1 ($filename) of type string is deprecated in /var/www/html/cacti/lib/installer.php on line 791
{"Mode":1,"Step":1,"Errors":[],"Eula":null,"Prev":{"Text":"Previous","Step":0,"Enabled":false,"Visible":true},"Next":{"Text":"Begin","Step":2,"Enabled":false,"Visible":true},"Test":{"Text":"Test Connection","Step":-4,"Enabled":true,"Visible":false},"Theme":"midwinter","StepData":{"Eula":null,"Theme":"midwinter","Language":"en-US"},"Html":"
```

## Screenshots

![Screenshot from 2022-12-04 18-05-19](https://user-images.githubusercontent.com/5182348/205505306-82af96ed-d839-4b68-b37c-6c44e7940f4c.png)

## Desktop (please complete the following information)

Ubuntu 22.04.1 LTS

**Comment by browniebraun** [comment]
More output generated within final step of the installation wizard:

`Deprecated: Implicit conversion from float-string "1670175096.0354" to int loses precision in /var/www/html/cacti/lib/installer.php on line 2654
{"Mode":null,"Step":"97","Errors":[],"Eula":"1","Prev":{"Text":"Previous","Step":96,"Enabled":true,"Visible":false},"Next":{"Text":"Next","Step":98,"Enabled":true,"Visible":false},"Test":{"Text":"Test Connection","Step":-4,"Enabled":true,"Visible":false},"Theme":"midwinter","StepData":{"Current":"41","Total":100},"Html":"`

**Comment by TheWitness** [comment]
Working.

**Comment by TheWitness** [comment]
Should be good now.

**Comment by netniV** [comment]
Ouch, you should really move forward from 8.1.2.  I thnk it's up to .13 now. :)

**Comment by netniV** [comment]
I believe that I fixed these warnings in 1.3.  It wasn't backported but looks like @TheWitness has covered it.  8.1/8.2 isn't fully supported in 1.2.x.

**Comment by TheWitness** [comment]
Found a few more in the installer based on check all pages this morning.
